### ISSUE TITLE
Top bar dropdowns have empty columns and missing borders

### ISSUE THREAD

**Issue by Woedenaz** [Nu-SCPTheme/Nu-SCP#8]
See attached image.

Need to find a way to make width decrease if there are less columns and add a bottom border to the last menu item.

![snipperino2](https://user-images.githubusercontent.com/46789324/51805838-6c325500-2240-11e9-8075-f03d75a69fc9.JPG)

**Comment by Woedenaz** [comment]
LOL WHOOPS I'll fix that. Sorry.

**Comment by rossjrw** [comment]
Border should be fairly easy to fix at some point, at least compared to the columns.

For the columns:

I tried a horizontal flex solution and a grid solution, using `nth-child(3n-x){ order: x }` and `nth-child(3n-x){ grid-column-start: x; }` respectively where x varied from 1 to 3 (for a 3 column layout) and worked out that we'd need a different value of x for each row of the grid, which is impossible, I'm pretty sure

I also found this http://jsfiddle.net/tcK3F/ , which I thought would allow me to select dropdown items based on their position in the list (the 1st third would go in the first column, the 2nd third would go in the middle column, etc), but I've got no idea how to adapt that for 3 or 4 columns.

I don't think we can lose the empty column without going back to horizontal ordering.
